storage/drive/drive.go

```diff
@@ -34,6 +37,17 @@ func New(folder string, service *drive.Service, l hclog.Logger) (storage.Storage
 		return nil, fmt.Errorf("failed to find folder: %w", err)
 	}
 	ds.p = f.Id
+	ds.gdcot = promauto.With(r).NewCounterVec(prometheus.CounterOpts{
+		Name: "ingest_google_drive_client_operations_total",
```

Comment:
i wonder if we should nor prefix with ingest and instead wrap the whole registry in the main ingest process with the prefix

Reply:
hmm yes this would be nice and fancy

Reply:
at the moment some metrics have the prefix but the go process collectors don't for example. Maybe in another PR though

Reply:
true. also it's a question if we actually want to prefix go process collectors, would be funny I think if we did